evm/src/executor/abi/mod.rs

```diff
@@ -12,184 +15,203 @@ pub const CHEATCODE_ADDRESS: Address = H160([
-            createSelectFork(string,uint256)(uint256)
-            createSelectFork(string,bytes32)(uint256)
-            createSelectFork(string)(uint256)
-            selectFork(uint256)
-            activeFork()(uint256)
-            transact(bytes32)
-            transact(uint256,bytes32)
-            makePersistent(address)
-            makePersistent(address,address)
-            makePersistent(address,address,address)
-            makePersistent(address[])
-            revokePersistent(address)
-            revokePersistent(address[])
-            isPersistent(address)(bool)
-            rollFork(uint256)
-            rollFork(bytes32)
-            rollFork(uint256,uint256)
-            rollFork(uint256,bytes32)
-            rpcUrl(string)(string)
-            rpcUrls()(string[2][])
-            rpcUrlStructs()(Rpc[])
-            parseJson(string)(bytes)
-            parseJson(string, string)(bytes)
-            parseJsonUint(string, string)(uint256)
-            parseJsonUintArray(string, string)(uint256[])
-            parseJsonInt(string, string)(int256)
-            parseJsonIntArray(string, string)(int256[])
-            parseJsonString(string, string)(string)
-            parseJsonStringArray(string, string)(string[])
-            parseJsonAddress(string, string)(address)
-            parseJsonAddressArray(string, string)(address[])
-            parseJsonBool(string, string)(bool)
-            parseJsonBoolArray(string, string)(bool[])
-            parseJsonBytes(string, string)(bytes)
-            parseJsonBytesArray(string, string)(bytes[])
-            parseJsonBytes32(string, string)(bytes32)
-            parseJsonBytes32Array(string, string)(bytes32[])
-            allowCheatcodes(address)
-            serializeBool(string,string,bool)(string)
-            serializeBool(string,string,bool[])(string)
-            serializeUint(string,string,uint256)(string)
-            serializeUint(string,string,uint256[])(string)
-            serializeInt(string,string,int256)(string)
-            serializeInt(string,string,int256[])(string)
-            serializeAddress(string,string,address)(string)
-            serializeAddress(string,string,address[])(string)
-            serializeBytes32(string,string,bytes32)(string)
-            serializeBytes32(string,string,bytes32[])(string)
-            serializeString(string,string,string)(string)
-            serializeString(string,string,string[])(string)
-            serializeBytes(string,string,bytes)(string)
-            serializeBytes(string,string,bytes[])(string)
-            writeJson(string, string)
-            writeJson(string, string, string)
-            pauseGasMetering()
-            resumeGasMetering()
-    ]"#,
+    "[
+        struct Log { bytes32[] topics; bytes data; }
+        struct Rpc { string name; string url; }
+        struct FsMetadata { bool isDir; bool isSymlink; uint256 length; bool readOnly; uint256 modified; uint256 accessed; uint256 created; }
```

Comment:
I tried formatting the structs to more lines but it doesn't seem like they get parsed? @mattsse

Reply:
yeah looks like the struct parser of abigen has limitations with new lines